### DISCUSSION TITLE
Filesystem for SharedMount Point /  KVM Over Fibre Channel on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by midhunpjos** [apache/cloudstack#9511] [Q&A] [answered]
I’m looking for recommendations on the best primary storage solution for KVM virtualization over Fibre Channel (FC). I need guidance on selecting the most suitable filesystem for mounting a Fiber Channel device on an Ubuntu  22.04 + KVM server, utilizing the "shared mount point" option within KVM.

Thank you

**Comment by BryanMLima** [comment, marked as answer]
Hey, @midhunpjos

I usually use OCFS2 for iSCSI and FibreChannel, and it works as expected. There are some guides on the internet that can help you set it up, but it shouldn't be that difficult.

**Reply by midhunpjos** [reply]
Thanks @BryanMLima . OCFS2 appears to be straightforward.
